src/main/kotlin/sg/com/quantai/etl/services/CryptoService.kt

```diff
@@ -0,0 +1,27 @@
+package sg.com.quantai.etl.services
+
+import com.fasterxml.jackson.databind.JsonNode
+import com.fasterxml.jackson.databind.ObjectMapper
+import org.springframework.stereotype.Service
+import org.springframework.web.client.RestTemplate
+import org.springframework.beans.factory.annotation.Value
+
+@Service
+class CryptoService(
+    private val restTemplate: RestTemplate,
```

Comment:
Use WebClient instead of restTemplate

Reply:
Actually, It was because I saw RestTemplate was being added into the ETLApplication.kt 
Hence, it was why I used it 
But I will use WebClient instead

Reply:
Done